.github/workflows/test.yml

```diff
@@ -52,3 +52,26 @@ jobs:
         run: yarn generate-api
       - name: Test bp-gallery
         run: yarn test:ci
+
+  test_bp_strapi:
+    name: Test bp-strapi
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./projects/bp-strapi
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
```

Comment:
Why do we still test on 2 different node versions? Didn't we decide to only use node 16 from now on?

Reply:
We didn't think much about this, just copied it from the react-tests 😅